src/app/Models/User.php

```diff
@@ -56,4 +56,17 @@ public function setPassword($password)
 
         return $this->save();
     }
+
+
+    /**
+     * Returns all the accounts belonging to this user.
+     *
+     * Their operations and corresponding operation types are fetched from the DB as well,
+     * as they are necessary to calculate tha balance of each account.
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\HasMany
+     */
+    public function accounts(){
+        return $this->hasMany(Account::class)->with('financialOperations.operationType');
```

Comment:
If I am not mistaken, this should be enough after adding the with property to FinancialOperation:
```suggestion
        return $this->hasMany(Account::class)->with('financialOperations');
```

Reply:
The same can be done for "with('financialOperations')" then, fully eliminating the 'with' clause from this command. There doesn't seem to be a situation where accounts should be fetched without their operations, so added the 'with' property to them as well.